src/server/frontend_xwayland/xwayland_connector.cpp

```diff
@@ -17,49 +18,40 @@
 
 #include "xwayland_connector.h"
 
-#define MIR_LOG_COMPONENT "xwaylandconnector"
 #include "mir/log.h"
 
-#include "mir/optional_value.h"
-#include "mir/terminate_with_current_exception.h"
 #include "wayland_connector.h"
 #include "xwayland_server.h"
 
 namespace mf = mir::frontend;
 
 mf::XWaylandConnector::XWaylandConnector(
     const int xdisplay,
-    std::shared_ptr<mf::WaylandConnector> wc,
+    std::shared_ptr<WaylandConnector> const& wc,
     std::string const& xwayland_path) :
-    enabled(!!wc->get_extension("x11-support"))
+    start_xwayland{!!wc->get_extension("x11-support") ?
```

Comment:
`!!` is unnecessary, as the `?` will force it into a bool anyway

Reply:
OK